### PR TITLE
chore: Example how to use arbitrary hidden component for setDragImage

### PR DESCRIPTION
**Pull request by mshabarov**
This documents a workaround while the https://github.com/vaadin/flow/pull/20490 is being investigated why it doesn't work on Mac.

### CODE REVIEW

File: articles/flow/create-ui/dnd/drag-source.adoc [hunk 1]
```diff
@@ -175,7 +175,22 @@ Any optional coordinates define the offset of the pointer location from the top
...
+
+*Example*: Use off-screen component as a drag image.
+[source,java]
+----
+Span dragImage = new Span("Drag Image Component");
+Style dragImageStyle = dragImage.getElement().getStyle();
+dragImageStyle.setPosition(Style.Position.ABSOLUTE);
+dragImageStyle.setTop("-100px");
+dragImageStyle.setLeft("-100px");
+dragImageStyle.setDisplay(Style.Display.NONE);
```
Comment (caalador) [inline articles/flow/create-ui/dnd/drag-source.adoc:191]: This part won't work without the patch that adds the display style change on drag start/end.
Offset image need to be visible and when having multiple images they should not be offset to the same spot.

Comment (mshabarov) [reply articles/flow/create-ui/dnd/drag-source.adoc:191]: Oh

Comment (mshabarov) [reply articles/flow/create-ui/dnd/drag-source.adoc]: Removed display none.

**Review by caalador — APPROVED** [review]

**Review by caalador — COMMENTED** [review]
Looks ok, but now with the feature in you can in the sample have `display: hidden` if you want as it will be toggled automatically.

**Review by caalador — APPROVED** [review]

**Comment by mshabarov** [comment]
Sure, added it back.

**Pull request merged**